blockchain/go.mod

```diff
@@ -20,6 +20,10 @@ require (
 	github.com/decred/dcrd/crypto/ripemd160 v1.0.1 // indirect
 	github.com/decred/dcrd/dcrec/edwards/v2 v2.0.2 // indirect
 	github.com/decred/slog v1.2.0 // indirect
+	github.com/klauspost/cpuid/v2 v2.0.9 // indirect
+	lukechampine.com/blake3 v1.2.1 // indirect
 )
 
 replace github.com/decred/dcrd/chaincfg/v3 => ../chaincfg
+
+replace github.com/decred/dcrd/wire => ../wire
```

Comment:
This replacement directive needs to be moved to commit "multi: Separate block hash and proof of work hash", otherwise `run_tests.sh` fails in the intervening commits.

Reply:
Nice catch.  I didn't notice because of my local workspace.  I also noticed the next commit needed to be tidied due to the addition of the new dep to `wire` as well.  All updated accordingly.